data_ingestion/03_Ingest_data_with_Redshift_v3.ipynb

```diff
@@ -0,0 +1,1956 @@
+{
```

Comment:
<p>Typos:</p><blockquote>you want <strong>ot</strong> make sure that the Redshift VPC </blockquote><blockquote> Your VPC should <strong>has</strong> the following</blockquote><p> </p><p>Embolden "yes" in this sentence: </p><blockquote>Your VPC should has the following two VPC attributes set to yes</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>